examples/magento-graphcms/components/Layout/LayoutNavigation.tsx

```diff
@@ -128,7 +130,20 @@ export function LayoutNavigation(props: LayoutNavigationProps) {
 
             <DesktopNavActions>
               {!router.pathname.startsWith('/search') && (
-                <SearchLink href='/search' aria-label={i18n._(/* i18n */ 'Search...')} />
+                <>
+                  {lgDown ? (
+                    <Fab href='/search' size='large' color='inherit'>
+                      <IconSvg
+                        aria-label={i18n._(/* i18n */ 'Search...')}
+                        src={iconSearch}
+                        size='large'
+                        sx={{ color: 'text.primary' }}
+                      />
+                    </Fab>
+                  ) : (
+                    <SearchLink href='/search' aria-label={i18n._(/* i18n */ 'Search...')} />
+                  )}
+                </>
```

Comment:
- useMediaQuery will result in a CLS so we can not use this, we should solve this with CSS.
- Can we move this logic into SearchLink instead of the examples directory?

Reply:
Changing the component would also result in it changing in the sidebar, would adding a prop for this be desired?

Reply:
And could you explain why using css instead of the mediaquery would not result in cls?

Reply:
> Changing the component would also result in it changing in the sidebar, which i assume is not desired. Would adding a prop for this be desired?

```
breakpoint?: Breakpoint
```

Like implemented in [LinkOrButton](https://github.com/graphcommerce-org/graphcommerce/blob/canary/packages/next-ui/Button/LinkOrButton.tsx)